ocpmodels/models/gemnet/gemnet.py

```diff
@@ -426,51 +428,16 @@ def select_edges(
     def generate_interaction_graph(self, data):
         num_atoms = data.atomic_numbers.size(0)
 
-        if self.use_pbc:
-            if self.otf_graph:
-                edge_index, cell_offsets, neighbors = radius_graph_pbc(
-                    data, self.cutoff, self.max_neighbors
-                )
-            else:
-                edge_index = data.edge_index
-                cell_offsets = data.cell_offsets
-                neighbors = data.neighbors
-
-            # Switch the indices, so the second one becomes the target index,
-            # over which we can efficiently aggregate.
-            out = get_pbc_distances(
-                data.pos,
-                edge_index,
-                data.cell,
-                cell_offsets,
-                neighbors,
-                return_offsets=True,
-                return_distance_vec=True,
-            )
-
-            edge_index = out["edge_index"]
-            D_st = out["distances"]
-            # These vectors actually point in the opposite direction.
-            # But we want to use col as idx_t for efficient aggregation.
-            V_st = -out["distance_vec"] / D_st[:, None]
-            # offsets_ca = -out["offsets"]  # a - c + offset
```

Comment:
@abhshkdz this line being commented out was correct right? Cause we do this for GemNet-OC

Reply:
Yeah, this seems consistent with the released version from last year: https://github.com/Open-Catalyst-Project/ocp/blob/65c2d6246e69169f43949858d39550d2a635c7e0/ocpmodels/models/gemnet/gemnet.py

Reply:
But shouldn't this logic be same for both GemNet-OC and GemNet-dT given the conventions are same?

Reply:
I haven't yet checked how these indices / offsets are used in GemNet-dT, but we'd break backward compatibility with released checkpoints if we change this.